algorithms/gr-dij/java/Dijkstra.java

```diff
@@ -0,0 +1,89 @@
+import java.lang.*;
+
+class Dijkstra {
+    // A utility function to find the vertex with minimum distance value,
+    // from the set of vertices not yet included in shortest path tree
+    static final int V = 9;
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6333211)